### PR TITLE
teams/tidb: a vote for guo-shaoge to TiDB maintainer

### PR DESCRIPTION
**Pull request by XuHuaiyu**

### CODE REVIEW

File: teams/tidb/membership.json [hunk 1]
```diff
@@ -2,14 +2,14 @@
     "_comment": "Items of lists in this file are in alphabetical order. Please keep it on modification.",
     "description": "",
     "maintainers": [
-        "XuHuaiyu",
```
Comment (bb7133) [inline teams/tidb/membership.json:5]: Please notice that we sort them in 'ASCII-alphabetical' order, so I think you don't need to reorder this.

Just like: https://github.com/pingcap/community/blob/b6895f306859e67b59a52b01b26262be43a3dbc8/teams/tidb/membership.json#L33-L41

**Review by bb7133 — COMMENTED** [review]
LGTM

**Review by bb7133 — APPROVED** [review]

**Review by zz-jason — APPROVED** [review]
LGTM

**Review by zimulala — APPROVED** [review]
LGTM

**Pull request merged**